homework-submission/week1/src/todo.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+class Item extends React.Component {
```

Comment:
Currently that's fine, but It is better to split up your components in different files for scalability.